101-privatelink-endpoint-sql/azuredeploy.json

```diff
@@ -0,0 +1,295 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "administratorLogin": {
```

Comment:
change the parameter names for the vm & sql so it's clear by the name which params are used for which resources

Reply:
done in 05b1977